src/dns_config/api/auth_nsg_api.py

```diff
@@ -86,6 +86,13 @@ def create(
         :return: Returns the result object.
         """ # noqa: E501
 
+        if len(self.api_client.configuration.default_tags) > 0 and body is not None:
```

Comment:
[to be removed]
This has been added as an example . If this looks alright , i will add this to the generator]

Reply:
can't this be done in a generic way? instead of adding it to each api separately?